src/linagora.esn.unifiedinbox/views/partials/email.pug

```diff
@@ -36,8 +36,20 @@
         message="email"
       )
 
-    .lv-footer.attachments(ng-if="email.attachments.length > 0")
-      inbox-attachment(ng-repeat="attachment in email.attachments", ng-if="::!attachment.isInline", attachment="::attachment")
+    div(ng-if="email.attachments.length > 0")
+      md-card
+        md-card-header(ng-click="isExpand = isExpand ? false : true")
+          md-card-header-text.flex.flex-start
+            i.mdi.mdi-paperclip(title="{{ attachmentsNumber > 1 ? 'Attachments' : 'Attachment' | translate }}")
```

Comment:
Maybe you can use `ngPluralize` to pluralize and format things

Reply:
Seems to be a good idea, though it seems ngPluralize is not used anywhere in the app.

Reply:
Yes, exactly @alagane and it works with a ternary so let keep it like this but interesting I was not aware of this directive!